### DISCUSSION TITLE
Special Splits

### DISCUSSION THREAD

**Discussion by rchampine** [ohmg-dev/OldInsuranceMaps#154] [General]
In some rare cases, Sanborn maps sheets indicate areas that are not to scale. In order to place buildings in the correct location, these sheets may require additional splitting that does not correspond with the typical black lines that separate maps.  For these rare cases, please post the resource link here so they can be tracked.
![image](https://github.com/mradamcox/ohmg/assets/136510277/08c7aaa0-fb22-468f-beea-8ba182379295)

**Comment by rchampine** [comment]
Sacramento, Calif. | 1952 | Vol. 1 p37
https://oldinsurancemaps.net/resource/8501

**Comment by rchampine** [comment]
San Francisco, Calif. | 1950 | Vol. 1
https://oldinsurancemaps.net/resource/8501
This volume and potentially all other San Francisco volumes have many sheets with right-of-ways that are not drawn to scale. Individual sheets with this issue will be tracked here when assigned within the University of Richmond georeferencing project.

**Reply by mradamcox** [reply]
It would be good to be able to add a volume-wide note about this somewhere in the interface, so people see it whenever they start working on SF content.

**Comment by nolancrees** [comment]
Des Moines, Iowa
1957 Volume 1a
At least one street, Cherry street, is drawn not-to-scale on sheets 10a-15a

**Comment by rchampine** [comment]
Richmond, Va. | 1952 | Vol. 5
https://oldinsurancemaps.net/resource/7512
This map has a driveway that's marked as not to scale because it's 100' short. Splitting to compensate.
![image](https://github.com/mradamcox/ohmg/assets/136510277/1d456941-b48d-4911-b353-2665fcc2d9df)

**Comment by rchampine** [comment]
Savannah, Ga. | 1955 | Vol. 2 p133
https://oldinsurancemaps.net/resource/13945
This sheet appears to omit a half block between 38th and 39th Streets W. These is no note for this section being out of scale, but adding a extra spit to compensate for this omission resulted in better georeferencing.

**Comment by rchampine** [comment]
Phoenix, Ariz. | 1949 | Vol. 1 p41
https://oldinsurancemaps.net/resource/10474
Added additional split to compensate for S. 3rd St, which is annotated as 80' wide, but does not appear to be drawn to scale compared to other rights of way on the sheet. Separating the blocks here results in better georeferencing.
<img width="605" alt="image" src="https://github.com/mradamcox/ohmg/assets/136510277/9a2862ec-ee89-47ec-ba5f-6627cbdcf4b1">

**Comment by mradamcox** [comment]
As we get closer to having a more comprehensive solution to this, and it's come up in other contexts recently, I have started a thread on the OHM/OIM forum as well: https://forum.openhistoricalmap.org/t/street-widths-not-to-scale-more-examples/389. It would be good to get some of these examples over there too.

**Reply by rchampine** [reply]
Noted! I will add some more examples soon.

**Comment by mhpob** [comment]
An analogous situation where a special split may be need, but not due to scale issues with the map:

Multiple sheets converge on a small street (20') that has multiple layers of addresses on either side, meaning that the addresses may cover each other up in the multi-mask. There are also bends in the sheet and comparatively little matching modern infrastructure, which makes it doubly-tough.

[Piney Road in this volume](https://oldinsurancemaps.net/map/sanborn09064_008), where pgs [385](https://oldinsurancemaps.net/document/81474) ,[386](https://oldinsurancemaps.net/layer/81717), and [387](https://oldinsurancemaps.net/layer/81721) intersect.

<img width="221" height="626" alt="image" src="https://github.com/user-attachments/assets/e76446a5-0978-4764-b78a-e48aaceed53a" />

**Reply by mhpob** [reply]
Copying over discussion in the OSMUS Slack to here:

@mhpob 
> A request for georef/mult-mask help!
https://oldinsurancemaps.net/map/sanborn09064_008
Usually if things are slightly off I can fudge it and call it a day, but Piney Road, at the intersection of pages 385, 386, and 387 in the map above (far top-left), is so thin that the road winds up fully-closed and/or the addresses overlaps. Would anyone mind taking a look at the GCPs of those three pages and the multi-mask to see what's off?

@jpole1
> I think I made things a little better. Maybe a tiny less accurate in regards to parcel lines, but better befitting the goal of a cleanly understood mosaic

@mradamcox 
> Hi guys, I don't usually recommend this because it can get pretty messy, but you could undo 385 and the chop the top right section into a different piece (and maybe the top left of 386?). That would allow you to condense the open space between Old Brook Rd and Piney Rd
